pphtml.py

```diff
@@ -265,7 +265,7 @@ def allTargetsAvailable(self):
         r = ["[pass] all target images in HTML available in images folder"]
 
         for line in self.wb:
-            m = re.search(r"[\p{L}-_\d]+\.(jpg|jpeg|png)", line)
+            m = re.search(r"[\p{L}-_\d]+\.(jpg|jpeg|png|svg)", line)
```

Comment:
Do we care about extension capitalization?

Reply:
I think it is in the guidelines somewhere that filenames should be lowercase, at least I haven't seen any with uppercase. Maybe pphtml should check for it, but I think it would be better as a future, more thorough enhancement.